src/main/java/org/mockito/internal/stubbing/InvocationContainerImpl.java

```diff
@@ -128,8 +129,8 @@ public void clearInvocations() {
         registeredInvocations.clear();
     }
 
-    public List<StubbedInvocationMatcher> getStubbedInvocations() {
-        return stubbed;
+    public List<Stubbing> getStubbedInvocations() {
+        return (List) stubbed;
```

Comment:
java generics are invariant, we need a wild card here: `public List<Stubbing> getStubbedInvocations() `